### DISCUSSION TITLE
Modifica orario di registrazione

### DISCUSSION THREAD

**Discussion by atlantisinformatica** [globaleaks/globaleaks-whistleblowing-software#4125] [Q&A] [answered]
Volevo sapere cortesemente se c'è la possibilità di modificare l'orario di registrazione, al momento risulta uno scarto di due ore indietro.

**Comment by evilaliv3** [comment, marked as answer]
Buongiorno, non si tratta di uno scarto di 2 ore, è il timezone del server che usa UTC come riferimento.

Come  ad esempio le 00:00 UTC sono le 2:00 in orario corrente italiano
